zebra-state/src/sled_state.rs

```diff
@@ -63,8 +88,11 @@ impl SledState {
         }
     }
 
-    /// Commit a finalized block to the state. It's the caller's responsibility
-    /// to ensure that blocks are committed in order.
+    /// Commit a finalized block to the state.
+    ///
+    /// It's the caller's responsibility to ensure that blocks are committed in
+    /// order. This function is called by [`process_queue`], which does, and is
+    /// intentionally not exposed as part of the public API of the [`SledState`].
     fn commit_finalized(&mut self, queued_block: QueuedBlock) {
         let QueuedBlock { block, rsp_tx } = queued_block;
```

Comment:
just tried seeing if this could be an inner method in `process_queue` but sadly rust doesn't support methods that aren't in the top level impl block :sob:.

Reply:
Yeah, I think this is already nested enough, since most of the work is already embedded in the transaction closure.